tests/js/environment.js

```diff
@@ -1,23 +1,28 @@
 var fluid = require("infusion");
 
-fluid.defaults("gpii.pouch.tests.environment", {
+fluid.defaults("gpii.test.pouch.environment", {
     gradeNames: ["fluid.test.testEnvironment"],
     port:       6792,
     baseUrl:    "http://localhost:6792/",
     testUrl:    "/sample/",
```

Comment:
This probably doesn't belong here. Should pick a single place, either the harness or the environment, for static configuration of this kind - and factor it off for your own tests.